src/script/components/Conversation/Conversation.tsx

```diff
@@ -460,6 +465,19 @@ export const Conversation = ({
     [addReadReceiptToBatch, repositories.conversation, repositories.integration, updateConversationLastRead],
   );
 
+  const onGoToLastMessage = () => {
+    if (activeConversation) {
+      activeConversation.initialMessage(undefined);
+      if (!activeConversation.hasLastReceivedMessageLoaded()) {
+        updateConversationLastRead(activeConversation);
+        activeConversation.release();
+        amplify.publish(WebAppEvents.CONVERSATION.SHOW, activeConversation, {});
+      } else {
+        rerenderMessageList();
```

Comment:
I think in this case we should just run a scroll animation and not reload the entire conversation. This would feel more natural